src/main/java/com/example/demo/services/LeaseManagementDropBoxImpl.java

```diff
@@ -100,4 +112,32 @@ public void dropboxCallback(Long leaseId) throws ApiException, EmptyResultDataAc
         //todo check resonse object
         log.info("lease status updated for leaseId: {}", leaseId);
     }
+
+    ZonedDateTime parseZonedDateTime(String dateStr) {
```

Comment:
I would make these functions private or put them into a utility class so you can use them somewhere else.